vexim/adminuserchangesubmit.php

```diff
@@ -146,7 +146,7 @@
       header ("Location: adminuser.php?failupdated={$_POST['localpart']}");
       die;
     }
-  } else if ($_POST['clear'] != $_POST['vclear']) {
+  } else if (!alias_validate_password($_POST['clear'],$_POST['vclear'])) {
```

Comment:
Also here.
Then please squash these commits into one, and we're good for merge!